tests/test_ocs_basic_install.py

```diff
@@ -190,10 +184,14 @@ def create_ebs_volumes(
 
 
 @retry((CephHealthException, CommandFailed), tries=20, delay=30, backoff=1)
-def ceph_health_check():
+def ceph_health_check(namespace=default.ROOK_CLUSTER_NAMESPACE):
     """
     Exec `ceph health` cmd on tools pod to determine health of cluster.
 
+    Args:
+        namespace (str): Namespace of of OCS (default:
```

Comment:
of of

Reply:
As I have already 2 approves and @vavuthu needs to rebase on this change I will merge now and will send new PR to fix this right after.  Thanks @RazTamir